image_generation/latent_consistency_model/OVLatentConsistencyModelPipeline.py

```diff
@@ -0,0 +1,240 @@
+from typing import Union, Optional, Any, List, Dict
+from diffusers.pipelines.stable_diffusion.safety_checker import (
+    StableDiffusionSafetyChecker,
+)
+from diffusers.pipelines.stable_diffusion import StableDiffusionPipelineOutput
+from diffusers.image_processor import VaeImageProcessor
+from diffusers import DiffusionPipeline
+import openvino as ov
+import torch
+
+class OVLatentConsistencyModelPipeline(DiffusionPipeline):
```

Comment:
I think this sample replicates what we have in Optimum-Intel. What is the value of this class then? Please note that both (Optimum version and this one) will require maintenance.

Reply:
@AlexKoff88, thank you for the detailed comments. Right now, the versions I've pushed in the draft PR are identical to the optimum samples - I'm working on isolating the HF dependencies step by step in the pipeline to incorporate the OV native runtime API. Per our chat, I will modify the snippets to show the native workflow of export from optimum -> compile, without instantiating the Pipeline classes. Thank you for the direction!